.gitignore

```diff
@@ -31,3 +31,5 @@ build/Release
 config/local.js
 
 .DS_Store
+
+views/layout.ejs
```

Comment:
I added this because my local layout.ejs file kept getting modified for some reason. If it's a problem I can add it to my local .gitignore file instead so that it doesn't keep getting committed.